tests/lib/cbprintf_fp/testcase.yaml

```diff
@@ -0,0 +1,69 @@
+common:
+  harness: console
+  platform_exclude: native_posix native_posix_64 nrf52_bsim
```

Comment:
can we add integration_platforms here with best platforms to build/run the tests to reduce load and number of tests being run in CI?

Reply:
@nashif Done, also rebased and fixed doxygen symbol sorting.